include/beman/net/detail/basic_socket_acceptor.hpp

```diff
@@ -4,6 +4,7 @@
 #ifndef INCLUDED_BEMAN_NET_DETAIL_BASIC_SOCKET_ACCEPTOR
 #define INCLUDED_BEMAN_NET_DETAIL_BASIC_SOCKET_ACCEPTOR
 
+#include "beman/net/detail/socket_base.hpp"
```

Comment:
I think the `#include` directives should consistently use `<file>`.
```suggestion
#include <beman/net/detail/socket_base.hpp>
```